modules.json

```diff
@@ -10,12 +10,11 @@
     "concertBidAdapter",
     "connectIdSystem",
     "consentManagementUsp",
-    "consentManagement",
+    "consentManagementTcf",
```

Comment:
Has been changed  from consentManagement to consentManagementTfc in [Prebid 9.0](https://github.com/prebid/Prebid.js/pull/11720)